website/src/components/slices/treeSlice.js

```diff
@@ -10,6 +10,10 @@ import { cacheFields } from '../../fieldsGenerator/cacheFields';
 import { serviceFields } from '../../fieldsGenerator/serviceFields';
 import { jobFields } from '../../fieldsGenerator/jobFields';
 import { coreLibField } from '../../fieldsGenerator/coreLibField';
+import axios from 'axios';
+import { downloadFile, toSnakeCase } from '../../utils/commonUtils';
+
+const BASE = 'http://127.0.0.1:5000/';
```

Comment:
do we have config files?

Reply:
read about it, we can store it in env variables, and to add that support to docusaurus we have to install a plugin for it, so should i go ahead with it?